internal/trie/node/decode_test.go

```diff
@@ -166,7 +166,8 @@ func Test_decodeBranch(t *testing.T) {
 			variant:          branchVariant.bits,
 			partialKeyLength: 1,
 			errWrapped:       ErrDecodeChildHash,
-			errMessage:       "cannot decode child hash: at index 10: reading byte: EOF",
+			errMessage: "cannot decode child hash: at index 10: unmarshalling []: decoding []byte: decoding " +
+				"length: decoding uint: reading byte: EOF",
```

Comment:
What the `unmarshaling []` wrapped context means? is it trying to unmarshal an empty slice?